packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx

```diff
@@ -152,6 +154,26 @@ const ChartConfigTabs: FC = () => {
                                     />
                                 );
                             })}
+                            {pivotDimension && (
+                                <>
+                                    <GridLabel>Stacking</GridLabel>
+                                    <GridLabel />
```

Comment:
forgotten tag ?

Reply:
<img width="464" alt="Screenshot 2022-03-28 at 17 02 37" src="https://user-images.githubusercontent.com/11660098/160439737-f43a3505-62ce-411f-b5de-c6d344dbef0c.png">

It was deliberate to force a new line. But I've removed it. The buttons are inline but this will change in the upcoming PR changing the layout tab